go.mod

```diff
@@ -185,7 +185,7 @@ require (
 	github.com/pierrec/lz4 v2.6.1+incompatible // indirect
 	github.com/pingcap/goleveldb v0.0.0-20191226122134-f82aafb29989 // indirect
 	github.com/pkg/browser v0.0.0-20180916011732-0a3d74bf9ce4 // indirect
-	github.com/pkg/errors v0.9.1 // indirect
+	github.com/pkg/errors v0.9.1
```

Comment:
I'm not sure where this package is directly referenced. Actually, we should use `github.com/pingcap/errors` instead.

Reply:
change it back to before